src/scss/components/_navigation.scss

```diff
@@ -0,0 +1,41 @@
+@use "../utilities/variables" as variables;
+
+nav, footer{
+    background-color: map-get($map: variables.$colors, $key: green-header-footer);
+    
+    footer{
+        min-height: 150px
+    }
+    
+    .navbar-nav{
+        align-items: flex-end;
+    }
+
+    .nav-link{
+        max-height: 50px;
+        border-radius: 15px;
+        align-self: center;
+        justify-self: center;
+    }
+
+    .nav-link:hover{
+        background-color: map-get($map: variables.$colors, $key:blue-light);
+    }
+
+    .disabled{
+        color: map-get($map: variables.$colors, $key: blue-dark);
+        background-color: map-get($map: variables.$colors, $key: green-background);
+    }
+}
+
+
+@media (min-width: 600px){
```

Comment:
Using media queries is heavily discouraged in this CA. Try to find the bootstrap way, using `container-md` for example.

Reply:
Fixed!